src/views/docs/en/tutorials/beyond-hello-world.md

```diff
@@ -0,0 +1,485 @@
+<!-- show the customer a receipt -->
+<h2>Thank you for your order!</h2>
+<p>
+  <strong>"${meal.name}"</strong> is on the way.<br>
+  Delivery: <code>${order.deliveryDate}</code><br>
+  Order id: <code>${order.orderID}</code>
+</p>
+  `
+
+  return { html: view }
+}
+
+exports.handler = arc.http.async(handler)
+```
+
+</div>
+</arc-tab>
+
+<arc-tab label="get-admin/">
+<h5>get-admin/index.js</h5>
+<div slot=content>
+
+```js
+// ./src/http/get-admin/index.js
+let arc = require('@architect/functions')
+let html = require('@architect/shared/html')
+
+async function handler(request) {
+  let today = new Date()
+  let tables = await arc.tables()
+  // query orders using the "ordersByDate" index to get today's orders
+  let todaysOrders = await tables.orders.query({
+    IndexName: 'ordersByDate',
+    KeyConditionExpression: 'deliveryDate = :today',
+    ExpressionAttributeValues: { ':today': today.toISOString().split('T')[0] }
+  })
+
+  let view = html`
+<h2>Todays Orders (${todaysOrders.Count})</h2>
+<table border=1 cellpadding=5>
+  <thead>
+    <tr>
+      <th>customer</th>
+      <th>meal</th>
+      <th>id</th>
+    </tr>
+  </thead>
+  <tbody>
+    ${todaysOrders.Items.map(o => `
+      <tr>
+      <td>${o.email}</td>
+      <td>${o.meal}</td>
+      <td>${o.orderID}</td>
+      </tr>
+    `.trim()).join('')}
+  </tbody>
+</table>
+  `
+
+  return { html: view }
+}
+
+exports.handler = arc.http.async(handler)
+```
+
+</div>
+</arc-tab>
+
+<arc-tab label="get-index/">
+<h5>get-index/index.js</h5>
+<div slot=content>
+
+```js
+// ./src/http/get-index/index.js
+let arc = require('@architect/functions')
+
+async function handler(request) {
+  // redirect to the menu
+  return {
+    statusCode: 301,
+    headers: { location: '/menu' }
+  }
+}
+
+exports.handler = arc.http(handler)
+```
+
+</div>
+</arc-tab>
+
+</div>
+</arc-viewer>
+
+## `@events` for pub/sub
+
+It's really nice to be able to act on events in another process so that our main HTTP functions aren't tied up doing other things that don't need to happen before returning HTML.
+
+For example, we may want to do something for each new order created. Use the `@events` pragma to create a named event our app can subscribe to with a new Lambda function. We'll add the following:
+
+```arc
+@events
+new-order
+```
+
+Again, running `npx arc init` is helpful to create a `.js` file at the correct location for our subscriber.
+
+### Publish an event
+
+Let's add a single line to our `post-orders/index.js` endpoint right after the order is saved: 
+
+```js
+// ./src/http/post-orders/index.js
+
+...
+  // publish a "new-order" event for background processing
+  await arc.events.publish({ name: 'new-order', payload: order })
+...
+```
+
+Super simple. Just specify the event name and use the order as the payload.
+
+### Subscribe to "new-order"
+
+To "listen" for these events, we'll update our fresh event function. Notice the `order` is passed directly as it was created to the handler function:
```

Comment:
`passed directly as it was created to the handler function` doesn't quite sound right when I say it out loud.